excerpt-kafka/src/main/java/org/apache/kafka/common/Configurable.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common;
```

Comment:
can we rename this package ?
it is very confusing.
IIUC you copied a few classes from the original kafka code.

Reply:
thank you for your feedback.
the excerpt module is merely an experiment. nothing depends on it yet. for now we depend on kafka libs directly.
so, if I understand you correctly, you don't mind extracting fragments of kafka code into our own special module (with renaming root package in order to avoid conflicts), do you?